### ISSUE TITLE
output.target vs. output.chunkFormat – why not documented properly ?

### ISSUE THREAD

**Issue by projektorius96** [webpack/webpack#15281]
**TL;DR** We should know what **target**s / **format**s we are dealing with at first glance , not somewhere in the middle of story of nowhere . Please adjust you documentation , making more digestible bite for "Getting started", webpack considered daunting exactly due to this. I am telling you this as junior-to-mid developer . This is honest review the contribution of webpack should pay their attention at .

---

With reference to the titled & what I've found having a look at source code , in my opinion, not very well documented . What I figured out should be properly explained one or the other way, as for example:

- `output.target` ( **default:web** [_if no browserlist installed_] | _[other values](https://webpack.js.org/configuration/target/)_) –  meant to denote the output (bundle) **format** ;
> NOTE : the `target` is relatively global configuration , may be overwritten by `output.chunkFormat`, unless the `.chunkFormat()` set to false (default) to accept global target configuration !

- 'output.chunkFormat' (**default:false** _other values:  'array-push' | 'commonjs' | 'module' | <any string>_) – **by default it listens for global of** `output.target` _unless specified other than default value_ .

---

In case I have misinterpreted something , please correct me , but with correspondence how things been documented till-up-date, I would not judge myself too much even if mistaken . It's always good to clarify . Thank you .
